markup/goldmark/convert_test.go

```diff
@@ -278,6 +278,15 @@ func TestConvertAttributes(t *testing.T) {
 				"table style",
 			},
 		},
+		{
+			"Code block, CodeFences=true,lineanchors",
+			func(conf *markup_config.Config) {
+				withBlockAttributes(conf)
+				conf.Highlight.CodeFences = true
+			},
+			"```bash {linenos=table, anchorlinenos=true, lineanchors=org-coderef--xyz}\necho 'foo';\n```",
+			"<div class=\"highlight\"><div style=\"color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;\">\n<table style=\"border-spacing:0;padding:0;margin:0;border:0;\"><tr><td style=\"vertical-align:top;padding:0;margin:0;border:0;\">\n<pre tabindex=\"0\" style=\"color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;\"><code><span style=\"white-space:pre;user-select:none;margin-right:0.4em;padding:0 0.4em 0 0.4em;color:#7f7f7f\" id=\"org-coderef--xyz-1\"><a style=\"outline: none; text-decoration:none; color:inherit\" href=\"#org-coderef--xyz-1\">1</a>\n</span></code></pre></td>\n<td style=\"vertical-align:top;padding:0;margin:0;border:0;;width:100%\">\n<pre tabindex=\"0\" style=\"color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;\"><code class=\"language-bash\" data-lang=\"bash\"><span style=\"display:flex;\"><span>echo <span style=\"color:#e6db74\">&#39;foo&#39;</span>;\n</span></span></code></pre></td></tr></table>\n</div>\n</div>",
```

Comment:
@bep I see that in other tests, you use `"table style"` instead of this verbose expected string. How do I convert this to use `"table style"`?

Thanks!

Reply:
I suspect what you want is `conf.Highlight.NoClass = true` (or something).

Reply:
I meang noClasses=false.

Reply:
Thanks. I have force-pushed a change.